sql/core/src/test/scala/org/apache/spark/sql/MathFunctionsSuite.scala

```diff
@@ -319,6 +370,18 @@ class MathFunctionsSuite extends QueryTest with SharedSparkSession {
       df.withColumn("value_brounded", bround('value)),
       Seq(Row(BigDecimal("5.9"), BigDecimal("6")))
     )
+    checkAnswer(
+      df.withColumn("value_rounded", round('value, 0, "up")),
+      Seq(Row(BigDecimal("5.9"), BigDecimal("6")))
+    )
+    checkAnswer(
+      df.withColumn("value_brounded", round('value, 0, "down")),
```

Comment:
qq: is there any reference of DBMSes that supports `down`, `up`, etc?

Reply:
sql-server does something similar : 
https://docs.microsoft.com/en-us/sql/t-sql/functions/round-transact-sql?view=sql-server-ver15

Reply:
Can we match the argument and usage with other references? I think we don't have to bother to come up with spark's own argument style.

Reply:
Could you please give me an example ?

Reply:
I currently don't - I haven't taken a look around other DBMSes on that.

Reply:
SAP Hana SQL ROUND function takes rounding mode as argument as we do : https://help.sap.com/viewer/7c78579ce9b14a669c1f3295b0d8ca16/Cloud/en-US/20e6a27575191014bd54a07fd86c585d.html

Reply:
ROUND(<number> [, <position> [, <rounding_mode>]])

Reply:
Hmm, there seems to be no standard / de-fact standard implementation of `round` which has the third argument.
As you mention, SAP Hana and MS SQL Server allow the third argument for `round` but they are still ununiform.

Reply:
yes